packages/airnode-deployer/src/utils/files.ts

```diff
@@ -60,11 +61,47 @@ export function parseReceiptFile(receiptFilename: string) {
     throw e;
   }
 
-  const validationResult = validateReceipt(receipt);
+  const validationResult = validateReceipt(receipt, nodeVersion);
   if (!validationResult.valid) {
     logger.fail('Failed to validate receipt file');
     throw new Error(`Invalid Airnode receipt file: ${JSON.stringify(validationResult.messages)}`);
   }
 
   return receipt as Receipt;
 }
+
+export function parseConfig(configPath: string, secrets: Record<string, string>): Config {
```

Comment:
I don't really get what's the reason for having two separate configuration parsing functions (one in `airnode-deployer` and one in `airnode-node`). Can you elaborate on why is that?

Reply:
I think it is because deployer also need to check for the cloud provider type, which is not checked in node. HOwever this check can be moved out of the parseConfig function. I'll implement it that way.